sandbox/cgroup/tracking.go

```diff
@@ -275,6 +274,19 @@ var doCreateTransientScope = func(conn *dbus.Conn, unitName string, pid int) err
 	mode := "fail"
 	properties := []property{{"PIDs", []uint{uint(pid)}}}
 	aux := []auxUnit(nil)
+	jobRemoveMatch := []dbus.MatchOption{
+		dbus.WithMatchInterface("org.freedesktop.systemd1.Manager"),
+	}
+	if err := conn.AddMatchSignal(jobRemoveMatch...); err != nil {
+		return fmt.Errorf("cannot subscribe to systemd signals: %v", err)
+	}
+	closeChan := make(chan struct{})
+	defer close(closeChan)
+	signals := make(chan *dbus.Signal, 10)
+	jobResultChan := make(chan string, 1)
```

Comment:
I'm entering an uncharted (for me) golang territory here, but wouldn't it be better if this channel transimtted `error` types?
Then you could send back an error if operations like `dbus.Store(...)` below fail, send `nil` if the Job status is `done`, and create another error otherwise.